app/models/agents/http_status_agent.rb

```diff
@@ -12,6 +12,7 @@ class HttpStatusAgent < Agent
 
     form_configurable :url
     form_configurable :disable_redirect_follow, type: :array, values: ['true', 'false']
+    form_configurable :changes_only, type: :array, values: ['true', 'false']
```

Comment:
@dsander You were right, ```:boolean``` works with existing settings.  Went with that for both controls.